src/qt/pivx/settings/settingsexportcsv.cpp

```diff
@@ -0,0 +1,207 @@
+// Copyright (c) 2019 The PIVX developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include "qt/pivx/settings/settingsexportcsv.h"
+#include "qt/pivx/settings/forms/ui_settingsexportcsv.h"
+#include <QFile>
+#include "csvmodelwriter.h"
+#include "guiutil.h"
+#include "optionsmodel.h"
+#include "qt/pivx/qtutils.h"
+#include "guiinterface.h"
+
+SettingsExportCSV::SettingsExportCSV(PIVXGUI* _window, QWidget *parent) :
+    PWidget(_window, parent),
+    ui(new Ui::SettingsExportCSV)
+{
+    ui->setupUi(this);
+
+    this->setStyleSheet(parent->styleSheet());
+
+    /* Containers */
+    ui->left->setProperty("cssClass", "container");
+    ui->left->setContentsMargins(10,10,10,10);
+
+    // Title
+    ui->labelTitle->setText(tr("Export Accounting"));
+    ui->labelTitle->setProperty("cssClass", "text-title-screen");
+
+    // Subtitle
+    ui->labelSubtitle1->setText(tr("Export your wallet's accounting data to a csv file."));
+    ui->labelSubtitle1->setProperty("cssClass", "text-subtitle");
+
+    // Location
+    ui->labelSubtitleLocation->setText(tr("Where"));
+    ui->labelSubtitleLocation->setProperty("cssClass", "text-title");
+    ui->labelSubtitleLocationAddress->setProperty("cssClass", "text-title");
+
+    ui->pushButtonDocuments->setText(tr("Select folder..."));
+    setCssProperty({ui->pushButtonDocuments, ui->pushButtonAddressDocuments}, "btn-edit-primary-folder");
+    setShadow(ui->pushButtonDocuments);
+
+    ui->pushButtonAddressDocuments->setText(tr("Select folder..."));
```

Comment:
This can possibly be also set in the `.ui` file, even though it isn't strictly a "static" string (gets changed when selecting a destination).